src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
+import * as Dom from './utils/dom.js';
 import * as Events from './events';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'lodash.assign';
```

Comment:
Switched to object.assign, not sure what the equivalent of `var assign = Object.assign || require('object.assign');` should look like with ES6 modules. Or if it matters to use the native one ever.